base-image/Dockerfile

```diff
@@ -66,8 +66,8 @@ USER ${NB_USER}
 WORKDIR ${HOME}
 
 # Install latest mambaforge in ${CONDA_DIR}
-RUN echo "Installing Miniforge..." \
-    && URL="https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh" \
+RUN echo "Installing Mambaforge..." \
+    && URL="https://github.com/conda-forge/miniforge/releases/latest/download/Mambaforge-Linux-$(uname -m).sh" \
```

Comment:
Probably should keep using Miniforge instead of Mambaforge which is discouraged since Sep 2023 (https://github.com/conda-forge/miniforge/pull/488). Doesn't look like they're deprecating Mambaforge anytime soon, but don't want to get caught up in that.

Reply:
good catch!